_episodes/02-shell-basics.md

```diff
@@ -1,31 +1,418 @@
+When you login to the cluster,
+you begin at the home directory:
+
+~~~
+[nelle@login001 ~]$ pwd
+~~~
+{: .bash}
+
+~~~
+/home/nelle
+~~~
+{: .output}
+
+You can list the files and folders
+in this directory using `ls`:
+
+~~~
+[nelle@login001 ~]$ ls
+~~~
+{: .bash}
+
+~~~
+genomics-workshop   genomics-workshop.zip
+~~~
+{: .output}
+
+> ## File and folder names
+> Notice that we used a hyphen (`-`) between the words
+> `genomics` and `workshop` for the folder `genomics-workshop`.
+> Always avoid using spaces in file and folder names.
+> To see why it's a bad idea, try running the following command:
+>
+> ~~~
+> [nelle@login001 ~]$ mkdir genomics workshop
+> ~~~
+>
+> Now, type `ls`. What do you see?
+>
+{: .callout}
+
+The `ls` command accepts many **switches**
+which modify its behaviour;
+try the following command:
+
+~~~
+[nelle@login001 ~]$ ls -a
+
+.bash_profile   .bashrc     genomics-workshop
+~~~
+
+The `-a` switch prints files/directories beggining with a dot (`.`).
+These are usually hidden from the output of `ls`.
+
+> ## Switches, inputs, parameters or arguments
+>
+> These are all names for similar things,
+> and often they are used interchangeably.
+> Anything that follows the name of a command entered into the shell
+> can be a switch, input, parameter or argument to that command.
+> Many commands can accept multiple switches
+> and compose them in meaningful ways.
+> Try the following `ls` commands:
+>
+> 1. `ls`
+> 2. `ls -a`
+> 3. `ls -F`
+> 4. `ls -l`
+> 5. `ls -F -a`
+> 6. `ls -Fa`
+> 7. `ls genomics-workshop`
+> 8. `ls -Fa genomics-workshop`
+>
+{: .callout}
+
+So far, we have been working in our "home" directories (`/home/username`).
+Let's change directories using `cd`:
+
+~~~
+[nelle@login001 ~]$ cd genomics-workshop
+~~~
+{: .bash}
+
+~~~
+[nelle@login001 genomics-workshop]$
+~~~
+{: .output}
+
+The prompt has changed to indicate our working directory.
+We can confirm this using `pwd`:
+
+~~~
+[nelle@login001 genomics-workshop]$
+~~~
+{: .bash}
+
+~~~
+/home/nelle/genomics-workshop
+~~~
+{: .output}
+
+> ## Exploring the `shell-basics` folder
+> 
+> This is a pen-and-paper exercise.
+>
+> Inside the `genomics-workshop` directory,
+> you will find the directory `shell-basics`.
+> Explore the directory `shell-basics`
+> using the commands you have learned so far,
+> and draw a diagram representing the
+> layout of the files and folders inside it.
+> Here is an example diagram:
+>
+> ~~~
+> ├── reminders.txt
+> ├── software/
+> │   ├── README.txt
+> │   └── matlab/
+> │       └── install.txt
+> └── thesis/
+> ~~~
+>
+> Compare your diagram with your neighbour's.
+> Did you find any differences?
+>
+> Here are some additional commands you may find useful
+>
+> Command           | Action
+> ------------------|--------------------------
+> `cd ..`           | Go "up" one directory
+> `ls DIR`          | List the contents of the directory `DIR`
+> `cd A/B`          | Change directory to `B`, a sub-directory of `A`
```

Comment:
I suggest a rewording:
"Change directory to `B`, a sub-directory of `A`"
to
"Change into a sub-directory of `A` by the name `B`"